tools/run-tests.py

```diff
@@ -185,8 +185,16 @@ def run_jerry_tests():
             break
 
         test_cmd = [TEST_RUNNER_SCRIPT, get_binary_path(job.out_dir), JERRY_TESTS_DIR]
+        skip_list = [];
+
+        if '--profile=es2015-subset' not in job.build_args:
+            skip_list.append("es2015\/")
+
         if script_args.skip_list:
-            test_cmd.append("--skip-list=" + script_args.skip_list)
+            skip_list.append(script_args.skip_list)
+
+        if len(skip_list):
```

Comment:
No need to get the length of the list. An empty list is False. So simply: `if skip_list:`